src/docs/Product/getRequesters.yaml

```diff
@@ -0,0 +1,36 @@
+paths:
+  /api/product/{productId}/requesters:
+    get:
+      tags:
+        - Product
+      summary: Shows IDs of all requesters of a product (Must be logged in)
+      parameters:
+        - in: path
+          name: productId
+          schema:
+            type: string
+          required: true
+          description: 'ID of the product'
+          examples:
+            'real ID of a deleted product':
+              value: '62ee5c280262c62f56124dfd'
```

Comment:
we use user with id 630167bafbcc20f4ac2d40b1 in swagger for login, so maybe we can add one of this user's product to see the actual requesters result, you can use this one if you like: 6303b1e98a23747a879d220c

Reply:
You pointed on some really important parts @okurezgi, thank you. Especially the populate and response parts were excellent. Saying that, unfortunately the FE will not be able to see some of these responses we're adding to the `.yaml` file, since the databases are different.
I'm still adding all your requests; they're solid points.